src/hooks/useAfterFirstRender/__stories__/useAfterFirstRender.stories.mdx

```diff
@@ -1,5 +1,6 @@
-import { Story, Canvas, Meta } from "@storybook/addon-docs";
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
 import useAfterFirstRender from "..";
+import { FunctionArgument, FunctionArguments, UsageGuidelines } from "../../../storybook";
```

Comment:
why? we add it to the configurations of storybook so you are not need to import it directly

Reply:
Don't know why, just some sense of perfectionism maybe... :D
Don't want to remove them rn - too many files, but I won't add them from now on, ok?